R/FilterStates.R

```diff
@@ -240,6 +240,10 @@ FilterStates <- R6::R6Class( # nolint
         )
       }
 
+      if (!is.null(private$module_add)) {
+        attr(fs, "module_add") <- private$module_add
+      }
```

Comment:
I think that attribute can be retained in the `FilteredData$get_filter_state` instead. We don't need to maintaing `private$module_add` in both classes.